src/scripts/gym/GymList.ts

```diff
@@ -195,36 +195,32 @@ GymList['Elite Lance'] = new Gym(
     [new GymBadgeRequirement(BadgeEnums.Elite_Agatha)]
 );
 // Kanto Champion
-GymList['Champion Blue'] = new Champion(
+GymList['Champion Blue'] = new Gym(
     'Blue',
     'Champion Blue',
     [
         new GymPokemon('Pidgeot', 52340, 59),
         new GymPokemon('Alakazam', 56320, 57),
         new GymPokemon('Rhydon', 58340, 59),
+        new GymPokemon('Exeggutor', 57520, 59, new StarterRequirement(GameConstants.Region.kanto, 0)),
```

Comment:
If we are going to be using these around, I think we should probably create something in `GameConstants` for each of the starters rather than just having magic numbers.

Reply:
Even if we just end up having something like:
```ts
declare enum starters {
  none = -1,
  grass = 0,
  fire = 1,
  water = 2,
  special = 3,
}
```